repository_creation/create_repository.go

```diff
@@ -61,7 +61,7 @@ func GenerateGithub(name, description string, ProjectType project_generation.Pro
 		Name:          github.String(repoName),
 		Description:   github.String(repoDescription),
 		DefaultBranch: github.String(defaultBranch),
-		MasterBranch:  github.String(masterBranch),
```

Comment:
Note: This is still 'MasterBranch' as this is the current structure of the field used by the API.